examples/vue/src/pages/ui/components/authenticator/sign-up-with-attributes/index.vue

```diff
@@ -21,6 +21,8 @@ Amplify.configure(aws_exports);
         placeholder="Zone Info"
         name="zoneinfo"
         :hideLabel="false"
+        :required="false"
+        :disabled="false"
```

Comment:
Remove these changes, as a reviewer it makes me question whether the changes impact the build artifacts